bundles/org.openhab.binding.thekeys/src/main/java/org/openhab/binding/thekeys/internal/api/model/LockersDTO.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.thekeys.internal.api.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Response DTO for /lockers endpoint
+ *
+ * @author Jordan Martin - Initial contribution
+ */
+public class LockersDTO {
```

Comment:
Are all you setters required ? They are never called.
Are they needed for the Json deserialization while your members are private ?

Reply:
I checked the GSON lib. It uses reflection and not the setters for deseriliazation. 
I removed the useless setters.